### PR TITLE
Radial bar chart

### PR DESCRIPTION
**Pull request by BugsBunny338**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: radial_bar_chart_plugin/.env [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+##
+
+# Specify backend URL including protocol. The Analytical Backend used by the development harness will connect
+# to this backend via proxy.
+#
+# If you do not provide backend URL, the harness will connect to public server used by GD.UI Examples Gallery
+#
+# Note: If you do provide backend URL, then you must also specify custom workspace and dashboard because the default
+# workspace and the default dashboard are only available on the public server.
+BACKEND_URL=https://jirizajic.demo.cloud.gooddata.com
```
Comment (kandl) [inline radial_bar_chart_plugin/.env:18]: Please remove these defaults

File: radial_bar_chart_plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
+# GoodData.UI Dashboard Plugin - Radial Bar Chart
+
+This plugin replaces all column charts with a custom [Recharts](https://recharts.org/) [radial bar chart](https://recharts.org/en-US/examples/SimpleRadialBarChart).
+
+This plugin was developed against https://jirizajic.demo.cloud.gooddata.com/, workspace id `ecommerce-parent`, dashoard id `c31f7bec-e820-4731-a451-974cd5d832c2`. This plugin should run against any dashboard on any workspace on any domain.
```
Comment (kandl) [inline radial_bar_chart_plugin/README.md:5]: Please describe how to setup this plugin (few steps, what to change, how and where)

File: radial_bar_chart_plugin/src/dp_radial_bar_chart_plugin/RadialBarChart.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+          .data()
+          .slices()
+          .toArray()
+          .map((slice, i) => ({
+            name: slice.sliceTitles()[0],
+            value: parseFloat(slice.rawData()[0]),
+            fill: indigoColors[i % indigoColors.length]
+          }));
+
+        console.log('result', data);
```
Comment (kandl) [inline radial_bar_chart_plugin/src/dp_radial_bar_chart_plugin/RadialBarChart.jsx:44]: Please remove all console logs

File: radial_bar_chart_plugin/src/dp_radial_bar_chart_plugin/RadialBarChart.jsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+        return (
+            <div style={{ height: '100%', width: '100%' }}>
+                <ResponsiveContainer height="99%">
+                    <RBC cx="50%" cy="50%" innerRadius="10%" outerRadius="100%" barSize={10} data={data}>
+                        <RadialBar
+                            minAngle={15}
+                            // label={{ position: 'insideStart', fill: '#fff' }}
```
Comment (kandl) [inline radial_bar_chart_plugin/src/dp_radial_bar_chart_plugin/RadialBarChart.jsx:52]: Please remove commented code

File: radial_bar_chart_plugin/src/harness/constants.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+// (C) 2021-2022 GoodData Corporation
+export const DEFAULT_WORKSPACE = "xms7ga4tf3g3nzucd8380o2bev8oeknp";
```
Comment (kandl) [inline radial_bar_chart_plugin/src/harness/constants.js:2]: please remove these defaults

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-plugin-examples-gate-PR29

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-sonarqube-scan-zuul-gate/27/ : SUCCESS in 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-merge-compliance-check/28/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-fossa-scanning-zuul/4/ : FAILURE in 2m 57s (non-voting)

**Pull request merged**